presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveCaching.java

```diff
@@ -82,6 +83,25 @@ private void testReadFromTable(String tableNameSuffix)
         query("DROP TABLE " + nonCachedTableName);
     }
 
+    /**
+     * Creates table with text files that are larger than 1MB
+     */
+    private String createTestTable(String tableName)
+    {
+        StringBuilder randomDataBuilder = new StringBuilder();
+        Random random = new Random();
+        for (int i = 0; i < 500_000; ++i) {
+            randomDataBuilder.append(random.nextInt(10));
+        }
+        String randomData = randomDataBuilder.toString();
+
+        query("DROP TABLE IF EXISTS " + tableName);
+        // use `format` to overcome SQL query length limit
+        query("CREATE TABLE " + tableName + " WITH (format='TEXTFILE') AS SELECT format('%1$s%1$s%1$s%1$s%1$s', '" + randomData + "') as col");
```

Comment:
nit: you can generate format using `5` so it is obvious that returned String matches what is inserted into table.